README.md

```diff
@@ -34,6 +34,9 @@ export NOTIFICATION_QUEUE_PREFIX='YOUR_OWN_PREFIX'
 export FLASK_APP=application.py
 export FLASK_DEBUG=1
 export WERKZEUG_DEBUG_PIN=off
+
+export NEW_RELIC_LICENSE_KEY=<ask for it>>
+export NEW_RELIC_APP_NAME=development.notify-api
```

Comment:
this'll mean your local celery instance will have the `development.notify-api` name 🤔 

i'd rather it had a separate app in newrelic for the tasks - not 100% sure of the best place to set that though, i'm not sure on how everyone calls celery locally

Reply:
Yeah I did realise this, but it's only locally, so ...

Reply:
It's going to be full of everyone's stuff anyway, I don't think - especially now when we're only exploring - we need to care about that local bucket much.

Reply:
good point about it merging everyones. i guess if people want to enable it they should probably end up defining their own system `leo.notify-admin` etc etc